health/health.d/tcp_listen.conf

```
@@ -74,8 +80,10 @@ component: Network
      warn: $this > 1
      crit: $this > (($status == $CRITICAL) ? (0) : (5))
     delay: up 10 down 5m multiplier 1.5 max 1h
-     info: average number of SYN requests was dropped due to the full TCP SYN queue over the last minute \
-           (SYN cookies were not enabled)
+     info: Average number of SYN requests was dropped due to the full TCP SYN queue over the last minute. \
+           TCP SYN queue is full. The system is dropping incoming TCP SYN requests. \
+           It may indicate SYN floods. \
+           To fix, enable SYN cookies.
```

```suggestion
           If you can determine that the traffic is legitimate, consider enabling SYN cookies.
```

The solution here is not to just blindly enable SYN cookies. They have their own issues, and won’t actually solve a SYN flood attack (just mitigate it). Other avenues of resolution should ideally be explored as well.